juriscraper/opinions/united_states/state/__init__.py

```diff
@@ -114,6 +114,16 @@
     "nyappterm_1st",
     "nyappterm_2nd",
     "nysupct",
```

Comment:
This is most of the content on those pages in the other courts.  I see that we have the commercial divisions going there but perhaps we should do something about that.